pom.xml

```diff
@@ -19,6 +19,11 @@
             <comments>All source code is under the MIT license.</comments>
         </license>
     </licenses>
+    
+    <properties>
+      <jenkins.version>1.625</jenkins.version>
+      <java.level>6</java.level>
+    </properties>
```

Comment:
I checked the version to use in [org.jenkins-ci.jenkins](https://github.com/jenkinsci/pom/blob/4a51f9fa6f129a46bec55817bd09bcba14d42ffd/pom.xml#L585)
But I didn't notice this was overriden by [org.jenkins-ci.main.pom](https://github.com/jenkinsci/jenkins/blob/stable-1.625/pom.xml#L624)

My bad. Updated now.